slack-base/src/main/java/com/hubspot/slack/client/models/blocks/messages/RichMessageBlock.java

```diff
@@ -0,0 +1,9 @@
+package com.hubspot.slack.client.models.blocks.messages;
+
+import com.hubspot.slack.client.models.blocks.Block;
+
+import java.util.List;
+
+public interface RichMessageBlock extends Block {
```

Comment:
I think that the block with the nested blocks is not a unique feature of message blocks. I'd probably renamed this to `HasNestedBlocks` or `BlockWithNestedBlocks`? Probably the former option is more common.

Reply:
Also, QQ, I remember there was some failure because of this additional type when we were not able to correctly parse a new type returned by Slack if it isn't mapped to a model in this client.
Is the same thing going to happen now?

Reply:
I think yeah, we can rename it to `BlockWithNestedBlocks`. 
The previous errors were related to  `MessageBlock`, which was more of a marker interface. 
I could not rely 100% on our test run, as we had only our services up, and it's not the complete data set to make a conclusion.
We definitely won't merge this PR today( gonna merge it the morning), so we will be able to catch an error if it happens again.

Reply:
@nliutyi-hs Can you try and add a nested block with an random unknown type(e.g. blabla) and run a unit test or something? I guess that's something we can easily catch even before we deploy any code.